### DISCUSSION TITLE
All Tools usage?

### DISCUSSION THREAD

**Discussion by ericrascon** [spdustin/ChatGPT-AutoExpert#71] [General]
The "All-tools" system prompt you created.

Am I supposed to enter that entire prompt at once or on separate occasions within the same conversation?

**Comment by veilm** [comment]
That is a system prompt that OpenAI wrote for their All Tools version of GPT-4. It's not supposed to be entered manually by the user.

**Comment by ericrascon** [comment]
I see! Do you know if the same applies to the other individual system prompts as well? Are they not supposed to be entered by the user?

**Reply by veilm** [reply]
The same applies to all of the prompts in https://github.com/spdustin/ChatGPT-AutoExpert/tree/main/_system-prompts. The [main README](https://github.com/spdustin/ChatGPT-AutoExpert/blob/main/README.md) has links to and explanations of the AutoExpert prompts, which are the ones Dustin wrote for users to use.

**Comment by ericrascon** [comment]
Thanks this makes a lot of sense!
